test/tests/providers/global/linkerd.deployment.rules.test.yml

```diff
@@ -0,0 +1,31 @@
+---
+rule_files:
+  - linkerd.deployment.rules.yml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: 'managed_app_deployment_status_replicas_unavailable{deployment="linkerd-destination", managed_app="destination", namespace="linkerd"}'
+        values: '_x5 0+0x10 1+0x20'
```

Comment:
I think your series is 10 minutes short

Reply:
Thanks, pushed [4d6cd82](https://github.com/giantswarm/prometheus-rules/pull/689/commits/4d6cd82633d0958f76cb5d6b05730d4caacaf8e3) before reading this